cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -108,28 +111,28 @@ func (d *driver) Shutdown() error {
 	if err := d.state.computeDomainManager.Stop(); err != nil {
 		return fmt.Errorf("error stopping ComputeDomainManager: %w", err)
 	}
-	d.plugin.Stop()
+	d.helper.Stop()
 	return nil
 }
 
-func (d *driver) NodePrepareResources(ctx context.Context, req *drapbv1.NodePrepareResourcesRequest) (*drapbv1.NodePrepareResourcesResponse, error) {
-	klog.Infof("NodePrepareResource is called: number of claims: %d", len(req.Claims))
-	preparedResources := &drapbv1.NodePrepareResourcesResponse{Claims: map[string]*drapbv1.NodePrepareResourceResponse{}}
+func (d *driver) PrepareResourceClaims(ctx context.Context, claims []*resourceapi.ResourceClaim) (map[types.UID]kubeletplugin.PrepareResult, error) {
+	klog.Infof("PrepareResourceClaims called with %d claim(s)", len(claims))
 
 	var wg sync.WaitGroup
 	ctx, cancel := context.WithTimeout(ctx, ErrorRetryMaxTimeout)
 	workQueue := workqueue.New(workqueue.DefaultControllerRateLimiter())
+	results := make(map[types.UID]kubeletplugin.PrepareResult)
 
-	for _, claim := range req.Claims {
+	for _, claim := range claims {
 		wg.Add(1)
 		workQueue.EnqueueRaw(claim, func(ctx context.Context, obj any) error {
-			done, prepared := d.nodePrepareResource(ctx, claim)
+			done, res := d.nodePrepareResource(ctx, claim)
```

Comment:
Nit: what was the motivation for renaming this? Is `preparedResult` make more sence?

Reply:
> what was the motivation for renaming this?

"prepared" to me has the notion of a boolean (which it isn't).

The very local scope in which this variable is used (six neighboring lines) justifies a short name, I almost called it `r`.